tests/system/test_integration.py

```diff
@@ -222,9 +222,17 @@ def test_transaction_index(self):
         self.load_docs_with_template(self.get_transaction_payload_path(name="payload.json"),
                                      'http://localhost:8200/v1/transactions',
                                      'transaction',
-                                     9,
+                                     4,
                                      query_index="test-apm-transaction-12-12-2017")
 
+    @unittest.skipUnless(INTEGRATION_TESTS, "integration test")
+    def test_span_index(self):
+        self.load_docs_with_template(self.get_transaction_payload_path(name="payload.json"),
+                                     'http://localhost:8200/v1/transactions',
+                                     'transaction',
```

Comment:
what do you think of adding another param to `load_docs_with_template` for `processor.event` and having the query check more specifically against `name` and `event`, instead of just `name`, as splitting will be on `event` once this goes in?

Reply:
I did [update the test](https://github.com/simitt/apm-server/blob/855fa9077c6a6aeb5fd4cb4e50ca98e61425be13/tests/system/test_integration.py#L204) slightly different - WDYT?

Reply:
lgtm, thanks